src/RectangleRoiMobileTool.js

```diff
@@ -70,13 +196,19 @@ export default class RectangleRoiMobileTool extends BaseTool {
     const element = evt.detail.element;
     const image = evt.detail.currentPoints.image;
 
+    // console.warn('_startOutliningRegion', this.isAnnotating)
```

Comment:
I leave it for now... as commented out code
it's important for refresh of drawing box

Reply:
maybe put in a comment in the code?